packages/form-field/src/FormField.spec.tsx

```diff
@@ -239,8 +365,10 @@ describe('packages/form-field', () => {
           </FormFieldInputContainer>
         </FormField>,
       );
-      const error = queryByText('This is an error message');
-      expect(error).toBeInTheDocument();
+      const errorIcon = container.querySelector('svg');
```

Comment:
same here, you could check for the `aria-label` or `title`

Reply:
updated!